sdk/textanalytics/ai-text-analytics/src/textAnalyticsClient.ts

```diff
@@ -354,7 +354,16 @@ export class TextAnalyticsClient {
         code: CanonicalCode.UNKNOWN,
         message: e.message
       });
-      throw e;
+      if (e.response.parsedBody.error.innererror.code === "InvalidDocumentBatch") {
+        const innerError = {
+          statusCode: e.statusCode,
+          code: e.response.parsedBody.error.innererror.code,
```

Comment:
A comment explaining why we're doing this will be helpful.

Reply:
done.